experimental/dds/tree2/src/test/rebase/rebaseBranch.spec.ts

```diff
@@ -0,0 +1,202 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert, fail } from "assert";
+import { validateAssertionError } from "@fluidframework/test-runtime-utils";
+// Allow importing from these specific files which are being tested:
+/* eslint-disable-next-line import/no-internal-modules */
+import { GraphCommit, RevisionTag, findCommonAncestor, rebaseBranch } from "../../core/rebase";
+import { NonEmptyTestChange, TestChange, TestChangeRebaser } from "../testChange";
+
+function node(
```

Comment:
I would prefer to call this something like `newCommit`.

Reply:
Fine with me. Are you sure you don't want to call it `ehiEhoBranch`?